src/backend/joanie/core/api/admin.py

```diff
@@ -221,6 +221,8 @@ def create(self, request, *args, **kwargs):
         data = request.data
         serializer = self.get_serializer(data=data)
         data["product"] = kwargs.get("product_id")
+        if data.get("course_runs", None) == "":
```

Comment:
More generally then:
```suggestion
        if not data.get("course_runs", None):
```